### ISSUE TITLE
feat: support several user accounts in the same phone

### ISSUE THREAD

**Issue by jgomer2001** [GluuFederation/super-gluu#25]
Currently it is not possible to enroll more than one account in the app. The second enrollment is considered as an auth attempt

**Comment by NazarYavornytskyy** [comment]
@jgomer2001 can you give more details: One/Two steps, iOS/Android?

**Comment by jgomer2001** [comment]
any step or platform. it's a general requirement

**Comment by NazarYavornytskyy** [comment]
So, for Two steps we can support multiple users in one server. But when One Step is activated in QR code we have those informations:

**Comment by jgomer2001** [comment]
not sure. Maybe @yurem and @nynymike can help here

**Comment by jgomer2001** [comment]
not sure. Maybe @yurem and @nynymike can help here

**Comment by nynymike** [comment]
I don't think we should over-engineer this feature. Perhaps what we should do is to just enable the end user to select a "default" key, which should be used in the case of one step authn.
